File: backend/backend/src/main/java/br/com/sbm/backend/model/Fruit.java
package br.com.sbm.backend.model;

import java.time.LocalDateTime;

public class Fruit {
	private Long id;
	private Integer quantity;
	private String origin;
	private LocalDateTime importDate;

	public Fruit() {
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public Integer getQuantity() {
		return quantity;
	}

	public void setQuantity(Integer quantity) {
		this.quantity = quantity;
	}

	public String getOrigin() {
		return origin;
	}

	public void setOrigin(String origin) {
		this.origin = origin;
	}

	public LocalDateTime getImportDate() {
		return importDate;
	}

	public void setImportDate(LocalDateTime importDate) {
		this.importDate = importDate;
	}
}

File: frontend/frontend/src/app/interface/fruit.ts
export interface Fruit {
  id?: number;
  quantity: number;
  origin: string;
  importDate: Date | string | undefined;
}

export interface FruitParams {
  quantity: number;
  origin: string;
  initialImportDate: Date | string | undefined;
  finalImportDate: Date | string | undefined;
}
